src/contact.cc

```diff
@@ -303,7 +303,7 @@ bool operator!=(
 std::ostream& operator<<(std::ostream& os, contact const& obj) {
   char const* cntctgrp_str(NULL);
   if (obj.contactgroups_ptr)
-    cntctgrp_str = chkstr(static_cast<contactgroup const*>(obj.contactgroups_ptr->object_ptr)->group_name);
+    cntctgrp_str = chkstr(static_cast<contactgroup const*>(obj.contactgroups_ptr->object_ptr)->get_name().c_str());
```

Comment:
chkstr is just here for null pointers. Here the name is a std::string, so never null. I think we can improve this code and maybe remove the cntctgrp_str pointer....

Reply:
sure, done